scenes/PlayerSelection.tscn

```diff
@@ -0,0 +1,710 @@
+[gd_scene load_steps=12 format=2]
+
+[ext_resource path="res://sprites/Background_gradient.png" type="Texture" id=1]
+[ext_resource path="res://sprites/Title_logo.png" type="Texture" id=2]
+[ext_resource path="res://sprites/Player_Orange.png" type="Texture" id=3]
+[ext_resource path="res://sprites/Player_Green.png" type="Texture" id=4]
+[ext_resource path="res://sprites/Player_Purple.png" type="Texture" id=5]
+[ext_resource path="res://sprites/Player_Yellow.png" type="Texture" id=6]
+[ext_resource path="res://sprites/Player_Red.png" type="Texture" id=7]
+[ext_resource path="res://sprites/Player_Blue.png" type="Texture" id=8]
+[ext_resource path="res://sprites/AI_Mark.png" type="Texture" id=9]
+[ext_resource path="res://sprites/Human_Mark.png" type="Texture" id=10]
+
+[sub_resource type="GDScript" id=1]
+script/source = "extends Node2D
+
+var PLAYER_SETTINGS = [
+	[ENUMS.PLAYER_TYPE.HUMAN, ENUMS.AI_STRATEGY.SOLO, false, false],
+	[ENUMS.PLAYER_TYPE.HUMAN, ENUMS.AI_STRATEGY.SOLO, false, false],
+	[ENUMS.PLAYER_TYPE.HUMAN, ENUMS.AI_STRATEGY.SOLO, false, false],
+	[ENUMS.PLAYER_TYPE.HUMAN, ENUMS.AI_STRATEGY.SOLO, false, false]]
+
+# ================================================== #
+# ================================================== #
+
+var standardButton = Color.gray
+var highlightedButton = Color.greenyellow
+
+var optionalButtonRed = Color.webgray
+var optionalButtonBlue = Color.webgray
+var optionalButtonGreen = Color.webgray
+var optionalButtonYellow = Color.webgray
+
+func object_features_init():
+	$HumanMarkRed.visible = true
+	$AIMarkRed.visible = false
+	$HumanMarkBlue.visible = true
+	$AIMarkBlue.visible = false
+	$HumanMarkGreen.visible = true
+	$AIMarkGreen.visible = false
+	$HumanMarkYellow.visible = true
+	$AIMarkYellow.visible = false
+	
+	$StartGameRect/Text.text = \"START GAME\"
```

Comment:
Its better to set the initial value of the text via property in the scene editor, not by the code, especially if this text is not meant to be ever changed